crypto/property/property.c

```diff
@@ -288,6 +289,26 @@ static int ossl_method_store_insert(OSSL_METHOD_STORE *store, ALGORITHM *alg)
     return ossl_sa_ALGORITHM_set(store->algs, alg->nid, alg);
 }
 
+/**
+ * @brief Adds a method to the specified method store.
+ *
+ * This function adds a new method to the provided method store, associating it
+ * with a specified NID, properties, and provider. The method is stored with
+ * reference count and destruction callbacks.
+ *
+ * @param store Pointer to the OSSL_METHOD_STORE where the method will be added.
+ * @param prov Pointer to the OSSL_PROVIDER representing the provider of the method.
+ * @param nid NID (numerical identifier) associated with the method.
```

Comment:
It's not a NID in the sense of the `NID_*` macros.  The parameter is not well named, I would just have named it `id` or `internal_id`.